### PR TITLE
Update desktop-environment-integration.md

### PR DESCRIPTION
**Pull request by PierBover**
I've added a little note for beginners (such as myself).

### CODE REVIEW

**Comment by felixrieseberg** [comment]
LGTM :+1:

File: docs/tutorial/desktop-environment-integration.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ to the user. Electron conveniently allows developers to send notifications with
 the [HTML5 Notification API](https://notifications.spec.whatwg.org/), using
 the currently running operating system's native notification APIs to display it.
 
+Note: since this is an HTML5 API it is only avaialable in the renderer process.
```
Comment (kevinsawicki) [inline docs/tutorial/desktop-environment-integration.md]: Minor typo here, shoudl be `available` instead of `avaialable`.

**Comment by PierBover** [comment]
Ooops... those damn gremlins! :smile:

It's fixed now.

**Comment by kevinsawicki** [comment]
:+1: :memo: Thanks for this

**Comment by PierBover** [comment]
My pleasure!

**Pull request merged**